plugins/woocommerce/tests/e2e-pw/tests/api-tests/customers/customers-crud.test.js

```diff
@@ -343,7 +375,7 @@ test.describe(
 
 				// Verify that the customer can no longer be retrieved.
 				const getDeletedCustomerResponse = await request.get(
-					`/wp-json/wc/v3/customers/${ customer }`
+					`./wp-json/wc/v3/customers/${ customerId }`
```

Comment:
This was a false negative. Because we were not sending the customerId the result was always a 404.